core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -303,10 +459,12 @@ class IndexingQueueActor(searchService: SearchService) extends Actor with ActorL
   // De-dupes files that have been updated since we were last told to
   // index them. No need to aggregate values: the latest wins. Key is
   // the URI because FileObject doesn't implement equals
-  var todo = Map.empty[String, FileObject]
+  private val todo = new m.HashMap[FileName, m.Set[FileObject]] with m.MultiMap[FileName, FileObject]
```

Comment:
This has to do with grouping all inner classes with their top level class to get scalasig out of it.